apps/asap-server/src/handlers/webhooks/webhook-connect-by-code.ts

```diff
@@ -24,7 +26,20 @@ export const handler: Handler = lambda.http(
     };
 
     const users = new Users();
-    await users.connectByCode(code, userId);
+    const squidex: Squidex<CMSUser> = new Squidex('users');
+    const user = await squidex
+      .fetchOne({
+        filter: {
+          path: 'data.connections.iv.code',
+          op: 'eq',
+          value: `ASAP|${code}`,
```

Comment:
This will not be backwards compatible with the codes created prior to this unless we try to match both new and old welcomeCode structures